src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -97,6 +97,10 @@ public function load(array $configs, ContainerBuilder $container)
             if (is_dir($dir = $bundle['path'].'/Resources/views')) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $name);
             }
+
+            if (null !== $bundle['parent']) {
+                $this->prependParentPaths($twigFilesystemLoaderDefinition, $container, $name, $bundle['parent']);
```

Comment:
Add a path in the user-configured paths for the same namespace, and ensure that it ends up as the first path in the loader config for this namespace